doc/whats_new/v1.3.rst

```diff
@@ -43,6 +43,12 @@ random sampling procedures.
   :user:`Jérémie du Boisberranger <jeremiedbb>`,
   :user:`Guillaume Lemaitre <glemaitre>`.
 
+- |Fix| :class:`decomposition.KernelPCA` now uses the correct `gamma` if `gamma` is
+  `None` every time the kernel is called. It should be equal to `1/num_features` of
+  the data that it is initially fitted on. As a consequence, the `KernelPCA` models
+  with `gamma=None` may produce different results through `inverse_transform`.
+  :pr:`26337` by :user:`Yao Xiao <Charlie-XIAO>`.
```

Comment:
Could you reword this to put the actual model change first? People may skim this section and might not catch what _actually_ changed